zookeeper/include/zk_nn_client.h

```diff
@@ -255,9 +255,17 @@ class ZkNnClient : public ZkClientCommon {
                      const std::string &path,
                      FileZNode &node);
   /**
-   * Given the filesystem path, get the full zookeeper path
+   * Given the filesystem path, get the full zookeeper path for the blocks
+   * where the data is located
    */
-  std::string ZookeeperPath(const std::string &hadoopPath);
+  std::string ZookeeperBlocksPath(const std::string &hadoopPath);
+
+   /**
+   * Given the filesystem path, get the full zookeeper path for the dir
+   * where the file metadata is written
+   */
+  std::string ZkNnClient::ZookeeperFilePath(const std::string &hadoopPath);
```

Comment:
i think you might not need the `ZkNnClient::` in front of this new function in this file.